### PR TITLE
Feature/4 progresspage2

### PR DESCRIPTION
**Pull request by sarah32-code**

### CODE REVIEW

**Comment by sarah32-code** [comment]
pull requrest!

File: progress.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            background-color: #253026;
+            color: white;
+        }
+
+        td {
+            text-align: center;
+        }
+
+        #progressTable {
+            margin-top: 20px;
```
Comment (mattchristenson) [inline progress.html:43]: this should be top and bottom
```suggestion
            margin-top: 20px;
            margin-bottom: 20px;
```

**Review by mattchristenson — COMMENTED** [review]
one change suggested

**Review by mattchristenson — APPROVED** [review]

**Comment by sarah32-code** [comment]
thanks!

**Pull request merged**